org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/codemanipulation/StubUtility.java

```diff
@@ -581,14 +606,8 @@ private static String evaluateTemplate(CodeTemplateContext context, Template tem
 		} catch (TemplateException e) {
 			throw new CoreException(Status.CANCEL_STATUS);
 		}
-		if (buffer == null) {
-			return null;
-		}
-		String str = buffer.getString();
-		if (Strings.containsOnlyWhitespaces(str)) {
-			return null;
-		}
-		return str;
+		String str = buffer == null ? null : buffer.getString();
+		return (str == null || str.trim().length() == 0) ? null : str;
```

Comment:
Strings.containsOnlyWhitespaces is more performant as it'll return when the 1st non whitespace character is met